funcframework/framework.go

```diff
@@ -86,9 +87,10 @@ func RegisterEventFunction(path string, fn interface{}) {
 
 // RegisterHTTPFunctionContext registers fn as an HTTP function.
 func RegisterHTTPFunctionContext(ctx context.Context, path string, fn func(http.ResponseWriter, *http.Request)) error {
```

Comment:
Hmm... I wonder if the logic for all this would be more straightforward if we just changed these functions to call registry.HTTP() and added a path field to `RegisteredFunction`...

Reply:
Done.